src/algebra/group/units.lean

```diff
@@ -398,16 +398,16 @@ noncomputable def is_unit.unit [monoid M] {a : M} (h : is_unit a) : Mˣ :=
 lemma is_unit.unit_of_coe_units [monoid M] {a : Mˣ} (h : is_unit (a : M)) : h.unit = a :=
 units.ext $ rfl
 
-@[to_additive]
+@[simp, to_additive]
 lemma is_unit.unit_spec [monoid M] {a : M} (h : is_unit a) : ↑h.unit = a :=
```

Comment:
I can see a weak argument for this one not being `simp`, as you lose the ability to apply lemmas about `units` once this is unfolded. Let's see whether CI cares.

Reply:
Yeah i also wondered about that, but I kinda think its analogous to subtype, where coe from a subtype simps to just the `val`?